packages/inferno/src/core/component.ts

```diff
@@ -82,16 +82,20 @@ function applyState<P, S>(component: Component<P, S>, force: boolean, callback?:
 
     renderCheck.v = true;
 
+    const parentDOM = (findDOMfromVNode(component.$LI, true) as Element).parentNode as Element;
+    const doc = (parentDOM || document).ownerDocument || document;
```

Comment:
parentDOM exists always so ` (parentDOM || document)` is unnecessary

Reply:
Im getting error from TSLINT if I remove this check - parentDOM is possible null - should I turn off tslint for this part or do you have any other idea?

Reply:
yeah its false lint error. You can do `(parentDOM as Element)` it cannot be null

Reply:
`
  ● Error recovery › Error recovery from user land errors › Error in child component › Should not block future updates - variation 2

    TypeError: Cannot read property 'ownerDocument' of null



      at applyState (packages/inferno/src/core/component.ts:2313:43)
      at rerender (packages/inferno/src/core/component.ts:2259:5)
      at Object.it (packages/inferno-create-element/__tests__/error.spec.jsx:624:9)
`

When I try to do it `const doc = (parentDOM as Element).ownerDocument as Document;` Im getting this error from test:coverage, any ideas what Im doing wrong?